cfme/common/vm.py

```diff
@@ -442,6 +442,31 @@ def unset_ownership(self):
         view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         view.flash.assert_success_message(msg)
 
+    def rename_vm(self, new_vm_name, cancel=False, reset=False):
```

Comment:
@ganeshhubale this method needs to be moved so that it is properly scoped.

If its only supported for vsphere vms, I agree with @izapolsk and @digitronik that it should exist in `InfraVm`, or even as a method in `VMwareProvider` that takes a vm object and a new name, returning a newly instantiated object with the new name.

Reply:
Also, this can just be called `rename()`, its already a method on the InfraVm class (should be ;) )